rules-tests/PHPUnit60/Rector/ClassMethod/AddDoesNotPerformAssertionToNonAssertingTestRector/Fixture/skip_do_test_file_info.php.inc

```diff
@@ -1,27 +0,0 @@
-<?php
-
-namespace Rector\PHPUnit\Tests\PHPUnit60\Rector\ClassMethod\AddDoesNotPerformAssertionToNonAssertingTestRector\Fixture;
-
-class SkipDoTestFileInfoTest extends \Rector\Testing\PHPUnit\AbstractRectorTestCase
-{
-    /**
-     * @dataProvider provideData()
-     */
-    public function test(\Symplify\SmartFileSystem\SmartFileInfo $fileInfo): void
-    {
-        $this->doTestFileInfo($fileInfo);
-    }
-
-    /**
-     * @return Iterator<\Symplify\SmartFileSystem\SmartFileInfo>
-     */
-    public function provideData(): \Iterator
-    {
-        return self::yieldFilesFromDirectory(__DIR__ . '/Fixture');
-    }
-
-    public function provideConfigFilePath(): string
-    {
-        return __DIR__ . '/config/symfony-26.php';
-    }
-}
```

Comment:
This was needed in back old days of Rector testing, method name hardcoded :). Not valid anymore and can be removed :+1:

Reply:
this is a fixture file with `.php.inc`, not a class that test a test.

Reply:
oh, I see, `doTestFileInfo` no longer exists 👍